src/ZoneServer/Skills/Handlers/Swordsman/Highlander/CrossGuard.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using Melia.Shared.Game.Const;
+using Melia.Shared.L10N;
+using Melia.Shared.World;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.World.Actors;
+
+namespace Melia.Zone.Skills.Handlers.Swordsman.Highlander
+{
+	/// <summary>
+	/// Handler for the Highlander skill Crown.
+	/// </summary>
+	[SkillHandler(SkillId.Highlander_CrossGuard)]
+	public class Highlander_CrossGuard : IGroundSkillHandler, IDynamicCasted
+	{
+		/// <summary>
+		/// Called when the user starts casting the skill.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		public void StartDynamicCast(Skill skill, ICombatEntity caster)
+		{
+			caster.StartBuff(BuffId.CrossGuard_Buff, skill.Level, 0, TimeSpan.Zero, caster);
+		}
+
+		/// <summary>
+		/// Called when the user stops casting the skill.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		public void EndDynamicCast(Skill skill, ICombatEntity caster)
+		{
+			caster.StopBuff(BuffId.CrossGuard_Buff);
+		}
+
+		/// <summary>
+		/// Handles skill, damaging targets.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="originPos"></param>
+		/// <param name="farPos"></param>
+		public void Handle(Skill skill, ICombatEntity caster, Position originPos, Position farPos, ICombatEntity target)
+		{
+			// Have to send this to prevent immobilization if you hold to max duration
```

Comment:
This comment is tricky. At first I thought you were documenting special behavior, but after thinking about it for a moment, this is just sent in place of other common response packets, like `ZC_SKILL_MELEE_GROUND`, isn't it? And since there are no attacks in here, the cast cancel must be sent to notify the client that it's over. That means I considered this something special because of the comment, when it probably isn't. That makes it not only another example of a type of comment we could put on most send calls, it may also raise incorrect assumptions about the code.

Reply:
I've somewhat moved the comment into the comment for the Handle method.